### PR TITLE
Peerreview

### PR DESCRIPTION
**Pull request by Ninjaphil24**
Hello Guys,

I tested to see if you can send all the commits in different branches and it works.  All the commits are saved on all the different branches.  Here's my final project ready for your review tomorrow morning.  Maybe you guys should create pull requests for yours as well, separate from what you'll send the reviewers.  Just do git checkout -b newname and that will save on a separate branch everything.  

See you tomorrow.

### CODE REVIEW

**Comment by souad988** [comment]
Hi @Ninjaphil24 

You did a really good job so far!
There are some small changes you need to do before you submit your work, but you are almost there!

To highlight:

✔ Meaningful commit messages.

Required Changes ♻️
Please check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

None.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: index.css [hunk 1]
```diff
@@ -6,43 +10,52 @@ body {
...
-  min-height: 100vh;
-  background-color: #e5e5e5;
+  background-image: url(images/Illustr.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-size: 50%;
+  display: grid;
+  grid-template-columns: 75px 1fr 1fr;
+  grid-template-rows: 1fr 10fr;
+  grid-gap: 20px;
```
Comment (souad988) [inline index.css:20]: it would be better for responsivenesse if you use % rather than px

File: index.css [hunk 2]
```diff
@@ -6,43 +10,52 @@ body {
...
 
 header .mobile {
   display: none;
 }
 
 header {
-  position: absolute;
-  width: 100%;
-  top: 0;
+  /* border: 1px solid yellow; */
```
Comment (souad988) [inline index.css:28]: [OPTIONAL] Please remove unused code, so your work looks clean

**Review by souad988 — COMMENTED** [review]
Hi @Ninjaphil24

You did a really good job so far!
There are some small changes you need to do before you submit your work, but you are almost there!

To highlight:

✔ Meaningful commit messages.

Required Changes ♻️
Please check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

None.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Comment by i-max-xi** [comment]
Okay @Ninjaphil24 great job 👍 
Really good work done.

Required Change ♻️ 
One tiny thing you could do to make it even more better is to update your README based on your commits.
You could also include screenshots of the deployed project into the README.
Again, great job @Ninjaphil24 👍

File: index.html [hunk 3]
```diff
@@ -53,90 +50,159 @@ <h3>I'm a Software Developer</h3>
...
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                    
+                </ul>
+            </div><!--greyout end-->
+            <div class="buttondecor">
+                <p>Not Active</p>
+            </div></a>
+        </div><!--site1 end-->
```
Comment (mwafrika) [inline index.html:134]: It should be good if you remove the commented code in your work, this is to help you to keep your code cleaned

File: index.html [hunk 4]
```diff
@@ -53,90 +50,159 @@ <h3>I'm a Software Developer</h3>
...
-        <div class="items buttontypeA">
+        <div class="buttontypeA">
             <a href="#">Get My Resume</a>
         </div>
-        <div class="items graphic">
-            <img src="images/graphic.png" alt="">
-        </div>
-        <div class="items hr">
+        <div class="hr">
             <hr>
```
Comment (mwafrika) [inline index.html:200]: kindly use prettier or any vscode extension to ident your code, It will help you organize your code better

**Review by mwafrika — COMMENTED** [review]
Congratulations @Ninjaphil24  
You are almost done, 
# Required changes ♻️ 
Please find the required changes in the comments below.

**Pull request merged**